index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello worlf!</h1[D[D[D[D[d[C[C[C[C[C>
```

Comment:
please, check
the word _world_ is written incorrectly and the code has strange symbols